packages/edit-widgets/src/components/layout/index.js

```diff
@@ -31,12 +27,21 @@ function Layout() {
 			>
 				{ areas.map( ( area, index ) => (
 					<div key={ index } className="edit-widgets-layout__area">
-						<WidgetArea title={ area } initialOpen={ index === 0 } />
+						<WidgetArea area={ area } initialOpen={ index === 0 } />
```

Comment:
Should we pass just an identifier here and use the data module in the subcomponent instead?

Reply:
Yes, we should 👍 I just used the simples way for now but that change will be applied as part of the polishing I'm doing.

Reply:
Should we implement this here?